resolwe_bio/tests/processes/test_annotation.py

```diff
@@ -0,0 +1,158 @@
+from pathlib import Path
+
+from django.contrib.auth.models import User
+
+from resolwe.flow.models import Data
+from resolwe.flow.models import Entity as Sample
+from resolwe.test import tag_process
+
+from resolwe_bio.utils.test import BioProcessTestCase
+from resolwe_bio.variants.models import Variant, VariantCall
+from resolwe_bio.variants.views import VariantAnnotationSerializer
+
+
+class AnnotationTestCase(BioProcessTestCase):
+    @tag_process("variant-annotation")
+    def test_variant_annotation(self):
+        with self.preparation_stage():
+            self.contributor = User.objects.get_or_create(
+                username="contributor_variants",
+                email="contributor_variants@genialis.com",
+            )[0]
+
+            self.sample = Sample.objects.create(contributor=self.contributor)
+
+            input_folder = Path("variant_annotation") / "input"
+            output_folder = Path("variant_annotation") / "output"
+            dbsnp = self.run_process(
+                "upload-variants-vcf",
+                {
+                    "src": input_folder / "dbsnp.vcf.gz",
+                    "species": "Homo sapiens",
+                    "build": "GRCh38",
+                },
+            )
+            clinvar = self.run_process(
+                "upload-variants-vcf",
+                {
+                    "src": input_folder / "clinvar.vcf.gz",
+                    "species": "Homo sapiens",
+                    "build": "GRCh38",
+                },
+            )
+            canonical_transcripts = self.run_process(
+                "upload-geneset",
+                {
+                    "src": input_folder / "canonical_transcripts_ens109.txt.gz",
+                    "source": "ENSEMBL",
+                    "species": "Homo sapiens",
+                },
+            )
+            dbsnp_2 = self.run_process(
+                "upload-variants-vcf",
+                {
+                    "src": input_folder / "dbsnp_2.vcf.gz",
+                    "species": "Homo sapiens",
+                    "build": "GRCh38",
+                },
+            )
+            clinvar_2 = self.run_process(
+                "upload-variants-vcf",
+                {
+                    "src": input_folder / "clinvar_2.vcf.gz",
+                    "species": "Homo sapiens",
+                    "build": "GRCh38",
+                },
+            )
```

Comment:
what is the purpose, what does this test?

Reply:
It tries to update the annotations of existing variant.